### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by pfagner**
Remove from the method an incorrect return that was present twice and returned an incorrect value

### CODE REVIEW

File: src/ipc/local/LocalIPCProxy.h [hunk 1]
```diff
@@ -160,7 +160,7 @@ class LocalIPCProxy : public IPCProxyBase<InterfaceType>, protected LocalIPCProx
     template<typename Type>
     using IPCProxyType = typename Type::IPCLocalProxyType;
 
-    LocalIPCProxy(QObject *parent = nullptr) : IPCProxyBase<InterfaceType>(parent), LocalIPCProxyBase(m_ipcBinder), m_ipcBinder(*this)
+    LocalIPCProxy(QObject *parent = nullptr) : IPCProxyBase<InterfaceType>(parent), LocalIPCProxyBase(*this), m_ipcBinder(*this)
```
Comment (jacky309) [inline src/ipc/local/LocalIPCProxy.h:163]: Make this part of a separate PR

**Comment by pfagner** [comment]
Deprecated commit.
